packages/fx-core/src/plugins/resource/function/constants.ts

```diff
@@ -182,16 +181,12 @@ export class QuestionValidationFunc {
 }
 
 export class FunctionBicep {
-  static readonly storageAccountName: string = "functionProvision.outputs.storageAccountName";
-  static readonly appServicePlanName: string = "functionProvision.outputs.appServicePlanName";
   static readonly functionEndpoint: string = "functionProvision.outputs.functionEndpoint";
 }
 
 export class FunctionArmOutput {
-  static readonly StorageName: string = "function_storageAccountName";
-  static readonly AppServicePlanName: string = "function_appServicePlanName";
   static readonly Endpoint: string = "function_functionEndpoint";
-  static readonly AppName: string = "function_appName";
+  static readonly AppId: string = "function_appResourceId";
```

Comment:
ResourceId? The name `AppId` seems confusing

Reply:
Fixed.